src/NotificationManager/index.js

```diff
@@ -0,0 +1,133 @@
+import React, {
+   createContext,
+   useContext,
+   useEffect,
+   useRef,
+   useState
+} from 'react'
+/** @jsx jsx */
+import { jsx } from '@emotion/core'
+import PropTypes from 'prop-types'
+import { createPortal } from 'react-dom'
+import { Transition, TransitionGroup } from 'react-transition-group'
+import OIOContext from '../OIOProvider/context'
+import { NotificationInline, View } from '..'
+
+const ANIMATION_DURATION = 600
+
+// =================================================
+// Context
+// =================================================
+
+const NotificationManagerContext = createContext()
+
+// =================================================
+// Provider
+// =================================================
+
+const NotificationManagerProvider = ({ children }) => {
+   const oioContext = useContext(OIOContext)
+   const notificationElem = useRef(document.createElement('div'))
+
+   useEffect(() => {
+      notificationElem.current.setAttribute('id', 'notification-container')
+      notificationElem.current.style.display = 'flex'
+      notificationElem.current.style.alignItems = 'flex-end'
+      notificationElem.current.style.position = 'absolute'
+      notificationElem.current.style.bottom = 0
+      notificationElem.current.style.width = '400px'
+      notificationElem.current.style.right = 0
+      notificationElem.current.style.pointerEvents = 'none'
+      notificationElem.current.style.fontFamily = oioContext.fontFamily
+      notificationElem.current.style.fontSize = oioContext.fontSize
+      document.body.appendChild(notificationElem.current)
```

Comment:
What happens if you have multiple `NotificationManager`s at the same time? Is the expected behaviour to have 2 containers? Or should 1 container be shared?